.github/workflows/docker.yml

```diff
@@ -0,0 +1,231 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - develop
+      - 'release/*'
+
+    tags:
+      - '*'
+
```

Comment:
I read about manual trigger github action using `workflow_dispatch`. Could you research a little bit on that?
I think you just need to add that to the "on:" section

https://dev.to/this-is-learning/manually-trigger-a-github-action-with-workflowdispatch-3mga#:~:text=There's%20a%20plethora%20of%20triggers,or%20create%20a%20pull%20request.